server/account.go

```diff
@@ -295,7 +295,7 @@ func (r *Router) AccountVerifyResend(c *fiber.Ctx) error {
 		return c.Render("account-verify-forgot-success.html", fiber.Map{})
 	}
 
-	if !user.Locked {
+	if !user.Locked && !viper.GetBool("accounts.require_admin_verify") {
```

Comment:
Curious if we need the check for `accounts.require_admin_verify` here? This particular endpoint allows a user to fill out form providing their username and they will be re-sent an email with a link to verify their email address. This particular check is to ensure only disabled/locked users will be re-sent the email. This modifies the logic such that: if a user is active AND accounts.require_admin_verify is true, then allow the verification email to get re-sent? What would be the use case for this? Their email/account is likely already verified and if they clicked the link again their account is already active? 

Do you envision a case where a user account is active but their email address is not verified?

Reply:
There is nothing stopping an admin to enable an account before a user has verified its email. Therefore I thought, it should be possible for a user to be able to be re-sent the verification email even if an admin has enabled its account.

Ideally, we would have two locks one for email verification and one for admin verification, and the account will be fully enabled only if both locks are unlocked. However, I could not find any mechanism in FreeIPA that would allow us to do that easily. A potential solution already discussed would be to create stage user, but that's a project for after Supercomputing 23.